internal/provider/signin_policy_resource.go

```diff
@@ -0,0 +1,249 @@
+package provider
+
+import (
+	"context"
+
+	"aembit.io/aembit"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ resource.Resource                = &SignInPolicyResource{}
+	_ resource.ResourceWithConfigure   = &SignInPolicyResource{}
+	_ resource.ResourceWithImportState = &SignInPolicyResource{}
+)
+
+// NewSignInPolicyResource is a helper function to simplify the provider implementation.
+func NewSignInPolicyResource() resource.Resource {
+	return &SignInPolicyResource{}
+}
+
+// SignInPolicyResource is the resource implementation.
+type SignInPolicyResource struct {
+	client *aembit.CloudClient
+}
+
+// Metadata returns the resource type name.
+func (r *SignInPolicyResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_signin_policy"
+}
+
+// Configure adds the provider configured client to the resource.
+func (r *SignInPolicyResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	r.client = resourceConfigure(req, resp)
+}
+
+// Schema defines the schema for the resource.
+func (r *SignInPolicyResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			// ID field is required for Terraform Framework acceptance testing.
+			"id": schema.StringAttribute{
+				Description: "This identifier is generated by Terraform provider",
+				Optional:    true,
+				Computed:    true,
+				// this is a static identifier to identify the SignInPolicy resource in the state,
+				Default: stringdefault.StaticString("signinpolicy_identifier"),
```

Comment:
Per my other comment, I'd recommend avoiding this static default and making it more dynamic.